tests/Console/EloquentCommandTest.php

```diff
@@ -41,7 +41,7 @@ public function testCommand()
         $tester = $this->runCommand($command);
 
         $expectedContent = '/**
- * 
+ *
```

Comment:
It seems the build broke due to this => https://travis-ci.org/github/barryvdh/laravel-ide-helper/builds/688241498

Please revert the change in this line

Reply:
I had a similar problem when adding a test case to the MethodTest.php. PHPStorm removes trailing whitespaces automatically, and I haven't found a way to make it stop this. However, I think it would be a better idea to explicitly put those whitespaces instead of hoping that code formatting will leave them in place. Thus, maybe better do

```
$expectedContent = "/**\n";
$expectedContent .= " * \n";
...
```

instead?

Reply:
> I had a similar problem when adding a test case to the MethodTest.php. PHPStorm removes trailing whitespaces automatically, and I haven't found a way to make it stop this. However, I think it would be a better idea to explicitly put those whitespaces instead of hoping that code formatting will leave them in place. Thus, maybe better do
> 
> ```
> $expectedContent = "/**\n";
> $expectedContent .= " * \n";
> ...
> ```
> 
> instead?
> 
> (I must admit, however, that I find it weird to test for whitespaces to trail some lines and not others in the first place...)

Great idea!

Reply:
Hmm. So, hello there, I came up with the tests in the first place, blame's on me 😄 

I was thinking about this for a second, but came to the following conclusion: if your editor is automagically trimming EOL spaces _inside_ string, you might want to reconsider this.

EOL whitespaces outside strings is insignificant, whereas inside, it is significant.

Of course you can fix the _generator_ here to not add the EOL whitespace which may end up in the actual code, but this should be a separate change.

Reply:
Hey! No blame :-)

You are absolutely right, it is odd that trailing spaces within multi line string literals are subject to modification in any way. However, a variety of IDEs seem to do this and my issue with the multi line string literal are not necessarily the whitespaces, but their existence in the first place. The phpdoc start (/**) does not have any trailing whitespaces, the next line (short description) does, the next line (long description) does not. Then all the parameters don't have trailing whitespaces, the return type does have a trailing whitespace, and so does the @static. The closing line then does not have a trailing whitespace again.

What I am trying to say is that it just seems a little inconsistent to me, and I think, just as you mentioned, the best way might be to get rid of all trailing whitespaces in phpdoc generation, as they don't really serve a useful purpose and are certainly not important enough to test for their existence...

Just a thought.

Reply:
FTR, I believe this now moot as https://github.com/barryvdh/laravel-ide-helper/pull/958 was merged basically all tests were switched to snapshots; I think this problem is now solved anyway

Reply:
> FTR, I believe this now moot as #958 was merged basically all tests were switched to snapshots; I think this problem is now solved anyway

Looking good :+1: